client/src/components/layouts/global.css

```
@@ -436,7 +436,17 @@ pre {
 
 .panel-primary > .panel-heading {
   background-color: transparent;
-  border-color: var(--color-primary);
+  border-color: var(--primary-color);
+  color: var(--primary-color) !important;
```

Do we still need the !important?

---

No, it does not appear necessary anymore.
```suggestion
  color: var(--primary-color);
```